core-bundle/assets/controllers/message-outlet-controller.js

```diff
@@ -13,6 +13,21 @@ export default class extends Controller {
         this._hide(event.target.closest('*[data-contao--message-outlet-target]'));
     }
 
+    renderMessage(event) {
+        const { type, message } = event.detail;
+
+        for (const target of this.messagePrototypeTargets) {
+            if(target.dataset.type === type) {
+                const html = target.getHTML().replace('{{message}}', message);
+                this.element.append(document.createRange().createContextualFragment(html));
+
+                return;
+            }
```

Comment:
```suggestion
            if (target.dataset.type === type) {
                this.element.append(document.createRange().createContextualFragment(target.getHTML().replace('{{message}}', message)));
                return;
            }
```
Could eliminate that const, right?

Reply:
Too long for single line imho.

Reply:
This will get compiled anyways.